cogctl/cli/profile.py

```diff
@@ -42,8 +42,9 @@ def create(state, name, host, port, secure, rest_user, rest_password):
     """
```

Comment:
Ah, I didn't realize they were subclasses of `click.Parameter`. I guess I should have scrolled up :)